### PR TITLE
[KC-2019] Initial agenda, bios, and abstracts

### PR DESCRIPTION
**Pull request by aaronblythe**
Thank you for allowing us a few hours to hack this together... rock and roll it on through.

We would like people on the team to comment on the initial schedule that comes out of the preview.

### CODE REVIEW

File: data/events/2019-kansas-city.yml [hunk 1]
```diff
@@ -404,4 +404,28 @@ program:
...
+  - title: "curtis-blackthorne"
+    date: 2019-10-24
+  - title: "jj-ashgar"
+    date: 2019-10-24
+  - title: "Ignite Karaoke!"
+    date: 2019-10-24
+  - title: "matt-blasinski"
+    date: 2019-10-25
+  - title: "kevin-reedy"
+    date: 2019-10-25
```
Comment (kevinreedy) [inline data/events/2019-kansas-city.yml]: I'll only be able to speak on 2019-10-24. This is in Paper Call as well.

Comment (aaronblythe) [reply data/events/2019-kansas-city.yml]: zoom meeting? I kid. Sorry for the miss. I will get that on the next push.

Comment (aaronblythe) [reply data/events/2019-kansas-city.yml]: https://github.com/devopsdays/devopsdays-web/pull/7954/commits/de633d06f76a38bae418842194d1fb54074fde67

**Comment by bridgetkromhout** [comment]
Hi, @aaronblythe - we appreciate you're looking to vet the PR across your team! We don't tend to keep PRs open for multiple days, though, so if it doesn't make sense to merge this today, we can close it and re-open when you're ready. Thanks!

**Comment by aaronblythe** [comment]
Understood. I believe we are done, other than the slots that are just a dash...
I am trying to figure that out, however I could use some help if this is something that is familiar to anyone else.

**Comment by mattstratton** [comment]
Also you prolly don't want the propose link in there anymore. I popped an edit into this PR for ya to fix that.

**Comment by aaronblythe** [comment]
So sorry for the delay... I knew there were two issues with my gem, but over the past two years I forgot (we used sessionize last year)... fixed that now: https://github.com/aaronblythe/papercallio/pull/6... lol... anyways... I will remove all the "do not merge" flags. You are free to go forward... we will likely have some fidgets over the next couple days.

**Pull request merged**